src/main/target/FOXEERF722DUAL/target.h

```diff
@@ -25,6 +25,7 @@
 
 #define ENABLE_DSHOT_DMAR       true
 #define LED0_PIN                PC15
+#define CAMERA_CONTROL_PIN      PB3
```

Comment:
Turns out the pin is already defined for this board in 3.5: https://github.com/betaflight/betaflight/pull/7248/files#diff-4f899a575af9b1f531cee162dc715497R34

(I **do** try to remember to add them when cherry-picking new target definitions into 3.5...)

Reply:
Ah,sorry i forgot it was define already below in foxeer target.I change it.